### ISSUE TITLE
Unable to add Plex account during setup

### ISSUE THREAD

**Issue by Spectre988** [PlexRipper/PlexRipper#200]
Newly installed and going through the setup.  When adding the Plex account screen is presented all info about my plex account is provided however the create button is not available.  

Running as a docker on Unraid. 

<img width="482" alt="image" src="https://user-images.githubusercontent.com/105654139/221307692-9efe6f10-4d1c-4620-a932-da5042c5e342.png">

**Comment by sudodrew** [comment]
do you have 2FA on? if so you need to add the code at the end of your password..

**Comment by craftpag** [comment]
Hello! 

Could you let me know what you typed in the username box?
when I created my plex account it was with Gmail. but plex doesn't know my Gmail password, so you must create a new password for your plex account in the settings menu. after you have created the password you can type in your Gmail account in the username box, password in the password box and display name in display name box. after you have filled all the boxes click on the check icon if everything is correct the CREATE box will turn green.

**Comment by Spectre988** [comment]
The username I'm using is my username for Plex.  I did try to switch to email but also not seeing the create button.  My plex account is not Gmail but....Hotmail.  Yeah I know...I am old.

![image](https://user-images.githubusercontent.com/105654139/224148302-29cf0387-4c57-4e3f-af0d-7183d2bb4c27.png)

**Comment by craftpag** [comment]
Ok, can you try to change your password inside plex? 
![plex](https://user-images.githubusercontent.com/94468418/224151481-87c233ed-3b76-42a6-9ff9-c4e637e8bbda.png)
Then use your Hotmail account and new password inside plexripper.
![Screenshot 2023-03-09 213537](https://user-images.githubusercontent.com/94468418/224151769-aa990364-067c-465a-a232-0b8f37f0a3c0.png)
what happens when you click on the check box left from the CREATE button

**Comment by craftpag** [comment]
Do you have a Plex server up and running?
what does the log say?
do to unraid docker menu and right-click on the plex ripper, then click on log and check for any errors.
if you find any errors send them here.

**Comment by Spectre988** [comment]
No errors.  How can I increase the debug level?

**Comment by craftpag** [comment]
Do you have media in your library? because you can only download media you already have. if you want new media you need an invite from another plex user. as far as i understand it, the purpose of plexripper is to allow plex users with large libraries to share their media with another user so they don't need to torrent it or loan a DVD from a friend. and to just download whatever you want from your own media library.

**Comment by Spectre988** [comment]
Yes I have media in my library.  

![Capture](https://user-images.githubusercontent.com/105654139/224738692-a8883f9c-c2fe-4a1b-b445-35c0a89fe876.JPG)

**Comment by JasonLandbridge** [comment]
@Spectre988, So the idea here is that you fill in your credentials and then click on the "Validate button":
 
![image](https://user-images.githubusercontent.com/15127381/225019200-32d67b65-5a3e-46e6-84da-cfe5489c4c8f.png)

Which will check if everything is correct before setting up the account. The fact that this was not clear for you is my fault so I will add a button text and maybe a description. If you have more ideas on how to make this more clear then please let me know. 

@craftpag, Thanks for answering the question! I realize now that based on your answer, that a Plex account can be created in multiple ways:

![image](https://user-images.githubusercontent.com/15127381/225020439-6475a068-17d1-439b-99d0-3d4703974c4c.png)

So that goes on the list of things to implement!

> The username I'm using is my username for Plex. I did try to switch to email but also not seeing the create button. My plex account is not Gmail but....Hotmail. Yeah I know...I am old.
> Also 2FA is not enabled.

Yeah username is I think meant to be your e-mail address, which is misleading by the fact is says "username". Will update that!

**Comment by JasonLandbridge** [comment]
> Do you have media in your library? because you can only download media you already have. if you want new media you need an invite from another plex user. as far as i understand it, the purpose of plexripper is to allow plex users with large libraries to share their media with another user so they don't need to torrent it or loan a DVD from a friend. and to just download whatever you want from your own media library.

This is indeed mostly correct, essentially PlexRipper connects to the media servers of others, which you have an invite for, and then it can download all movies and tv shows. 

@Spectre988, I have made a new release ([0.10.0-RC-1](https://github.com/PlexRipper/PlexRipper/releases/tag/v0.10.0-RC1)), available on both the [latest](https://hub.docker.com/r/plexripper/plexripper/tags) and [dev](https://hub.docker.com/r/plexripper/plexripper/tags) docker channels, which contains all the changes/fixes from the past several months. Let me know if this fixes the issues you have been facing

**Comment by Spectre988** [comment]
> @Spectre988, So the idea here is that you fill in your credentials and then click on the "Validate button":
> 
> ![image](https://user-images.githubusercontent.com/15127381/225019200-32d67b65-5a3e-46e6-84da-cfe5489c4c8f.png)
> 
> Which will check if everything is correct before setting up the account. The fact that this was not clear for you is my fault so I will add a button text and maybe a description. If you have more ideas on how to make this more clear then please let me know.
> 
> @craftpag, Thanks for answering the question! I realize now that based on your answer, that a Plex account can be created in multiple ways:
> 
> ![image](https://user-images.githubusercontent.com/15127381/225020439-6475a068-17d1-439b-99d0-3d4703974c4c.png)
> 
> So that goes on the list of things to implement!
> 
> > The username I'm using is my username for Plex. I did try to switch to email but also not seeing the create button. My plex account is not Gmail but....Hotmail. Yeah I know...I am old.
> > Also 2FA is not enabled.
> 
> Yeah username is I think meant to be your e-mail address, which is misleading by the fact is says "username". Will update that!

Yes, I am now able to login with my Plex account info but I keep getting the error of inspecting accessible servers.

**Comment by Spectre988** [comment]
![image](https://user-images.githubusercontent.com/105654139/225326208-f229d7c2-e715-454b-add7-3bd3696f4185.png)

**Comment by JasonLandbridge** [comment]
Are you on the latest version 0.10.0-RC1? And it might take 5 minutes to inspect the servers, just wait a little while

**Comment by Spectre988** [comment]
![image](https://user-images.githubusercontent.com/105654139/225411332-866682b1-08b3-4933-b5b5-6db8f8bdcfe8.png)


Seems not but the dockers says it is up to date.

**Comment by JasonLandbridge** [comment]
Are you on Unraid? Or on what platform is PlexRipper running because your version is indeed outdated

**Comment by Spectre988** [comment]
Unraid

**Comment by JasonLandbridge** [comment]
At the bottom of the Docker screen you can check for updates:

![image](https://user-images.githubusercontent.com/15127381/225623599-98b6e214-c15a-45ee-949c-1f0b8ff192c5.png)

And then something like this would show up next to PlexRipper

![image](https://user-images.githubusercontent.com/15127381/225623698-ca000e8f-3c67-4b6d-8c55-8200d19311f4.png)


Before doing this, make sure to stop the PlexRipper container and then clear the `/config` folder of any files. These will be recreated when PlexRipper starts

**Comment by Spectre988** [comment]
![image](https://user-images.githubusercontent.com/105654139/225629240-f4f8ba67-df8f-44ad-8c59-bd88199eb7ff.png)

**Comment by Spectre988** [comment]
I can see my server as well.  Also AFTKMST12 & AFTMM
